tools/clang/test/HLSLFileCheck/disassembler/amplificationtest.hlsl

```diff
@@ -1,6 +1,6 @@
 // RUN: %dxc /T as_6_6 /E ASMain %s | FileCheck %s
 // CHECK: Amplification Shader
-// CHECK: Numthreads: (2,4,1)
+// CHECK: NumThreads=(2,4,1)
```

Comment:
I don't prefer the formatting, but I suppose it's consistent with what was already there. So let's be consistent.